migrations/2_deploy_contracts.js

```diff
@@ -1,6 +1,10 @@
 const MyToken = artifacts.require("./MyToken.sol")
+const DSToken = artifacts.require("./DSToken.sol")
 
 module.exports = async function (deployer) {
+  await deployer.deploy(DSToken, "DAI")
+  const dai = await DSToken.deployed()
```

Comment:
You're assigning this to `dai`, but you never use that variable anywhere.

Reply:
Deleted